packages/app/src/cli/services/dev/processes/dev-session/dev-session.ts

```diff
@@ -221,6 +221,22 @@ export class DevSession {
       await this.logger.debug('❌ App preview update aborted (new change detected or error during update)')
     } else if (result.status === 'remote-error' || result.status === 'unknown-error') {
       await this.logger.logUserErrors(result.error, event?.app.allExtensions ?? [])
+
+      // Check for app uninstallation error
+      if (result.status === 'remote-error') {
+        const errors = result.error
```

Comment:
If `remote-error` and `unknown-error` no longer do the same thing (so you need this extra `if` here), let's split them in different `else-ifs`, that way you don't need to check for `remote-error` twice